langlib/lang.__internal/src/main/java/org/ballerinalang/langlib/internal/SelectDescendants.java

```diff
@@ -47,7 +49,10 @@ public static BXml selectDescendants(BXml xml, BString[] qnames) {
                 }
```

Comment:
Small improvement regarding the existing code. Shall we update this for loop in this manner
 `for (BString qname : qnames) {`